.docker/entrypoint/spec/jekyll_config_provider_spec.rb

```diff
@@ -39,6 +39,10 @@
         it {
           is_expected.to include('incremental' => true)
         }
+
+        it {
+          is_expected.to include('livereload_port' => 35729)
```

Comment:
Codacy found an issue: [Use underscores(_) as thousands separator and separate every 3 digits with them.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=5780130)